### ISSUE TITLE
DIM12-BUTTONS DO NOT HAVE NAME

### ISSUE THREAD

**Issue by Anitta-31** [Anitta-31/Dimcheck#12]
### Bug type

Other

### What happened?

When a button doesn't have an accessible name, screen readers and other assistive technologies announce it as button, which provides no information to users about what the button does.

<img width="273" alt="Screen Shot 2021-12-23 at 15 03 15" src="https://user-images.githubusercontent.com/93282832/147244729-0a8aea98-9648-4387-b2e5-be6e183f0460.png">

### Expected behavior

For buttons use attribute to clearly describe the action to anyone using.
The button that is used ,shoud have a name that is used for.
Example: back button ,delete or edit something.

### Reproduction link

https://staging-app.dimcheck.com/exceptions

### Reproduction steps

The problem should be resolved everywhere.

### Version (bug)

X.X

### What browsers are you seeing the problem on?

Chrome

### On what operating system are you experiencing the issue?

Mac OSX

### Pull Request

- [ ] I would like to do a Pull Request

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
